src/main/java/uk/gov/hmcts/reform/pip/rules/model/Publication.java

```diff
@@ -18,6 +18,6 @@
     /**
```

Comment:
I don't believe this class is used any more so can be deleted

Reply:
deleted